playbook.yml

```diff
@@ -70,6 +61,7 @@
       loop:
         - namespace.yaml.j2
         - operatorgroup.yaml.j2
+        - catalogsource.yaml.j2
```

Comment:
this should probably be a catalogsourceconfig...not catalogsource

Reply:
thank you, I've made the change.